docs/EXAMPLES.md

```diff
@@ -0,0 +1,517 @@
+Examples
+===============================
+
+> Notes:
+>
+> 1. All the examples are using ES6 syntax;
+>
+> 2. All the arguments in the tables are in the order that they must be passed in the method;
+>
+> 3. The `timeoutInMilliseconds` default is 5000 milliseconds;
+>
+> 4. If you pass the `timeoutInMilliseconds` argument, the timeout is changed only in the respective method. Except on the method `setTimeout`, that changes the timeout of all methods;
+>
+> 5. `timeoutInMilliseconds` turns mandatory if you need to change the `errorMessage`, due to arguments order;
+>
+> 6. If the `errorMessage` argument is not provided a default message implemented for this specific method will be displayed instead. We recommend that you use the default message if possible, because they show a clear message.
+
+## setTimeout
+
+| 1 optional argument |
+|:-----------:|
+|`timeoutInMilliseconds` (default: 5000)|
+
+```js
+const protractorHelper = require("protractor-helper");
+
+describe("foo", () => {
+    it("bar", () => {
+        // ...
+        // Here all `protractor-helper` method use the default timeout.
+        
+        protractorHelper.setTimeout(1000);
+
+        // ...
+        // Here all `protractor-helper` method use 1000 ms how timeout.
+        // ...
+        
+        protractorHelper.setTimeout();
+
+        // Here all `protractor-helper` method use the default timeout.
```

Comment:
Shouldn't it be `// Here all the protractor-helper methods use the default timeout.`?

Reply:
Commit made because this review: e59629c